### ISSUE TITLE
Mac gui stopped working (macos, osx)

### ISSUE THREAD

**Issue by Nadav-Ruskin** [debauchee/barrier#1920]
### What happened?

GUI has stopped working. Launching the barrier app doesn't show open any window. It has stopped working a few months ago. Reinstalling barrier, from homebrew and from the github release, has not helped. Nuking my old configuration has not helped.

This is a big problem because I'm not sure how to allow a fingerprint without the GUI... 

### Version

v2.4.0

### If applicable, where did you install Barrier from?

github, tried homebrew too

### What OSes are you seeing the problem on? (Check all that apply)

macOS

### What OS versions are you using?

macOS 13.2.1 (22D68)

### Relevant log output

```shell
I'm not sure where the mac log outputs to actually, couldn't find it in the docs.
```

### Any other information

I'd be grateful if anyone could help me with the following workarounds:

* Adding fingerprints (naively adding the fingerprint to files found in `find / -type f -name '*Local.txt' 2>/dev/null` did not help)
* Setting up barrierc to launch at startup, or alternatively the config location to the barrier app because I can get to to launch at startup but I can't access the GUI

**Comment by Abh15h3k** [comment]
Facing the same issue.

A workaround that i found works is launching the barrier executable from the terminal.
from the terminal execute the following
`/Applications/Barrier.app/Contents/MacOS/barrier`

only caveat you have to keep the terminal running. if the terminal exits barrier shuts down.

**Comment by tovoro** [comment]
Same issue here. When I try to start to start the barrier app, it always asks for the security access rights and quits again.

**Comment by ylluminate** [comment]
AFAIK, the project has moved to: https://github.com/input-leap/input-leap

**Comment by Nadav-Ruskin** [comment]
Thank you @ylluminate . Looks like that project is in a "build it yourself" phase... will try

**Comment by ylluminate** [comment]
Yep, seems like the Homebrew release is working alright on macOS. I actually helped someone setup the macOS Homebrew formula on an old MacPro running OCLP and a Windows version running this Barrier version and they were still compatible without any seeming issues. I simply disabled encryption since they operate on a very isolated network and they didn't care about privacy/security.
